package.json

```diff
@@ -12,8 +12,8 @@
   "readme": "./README.md",
   "scripts": {
     "start": "node .",
-    "dev": "npm-run-all --parallel watch:app watch:scss watch:js watch:translations",
-    "watch:app": "NODE_ENV=development nodemon .",
+    "dev": "npm-run-all --parallel watch:app watch:scss watch:js",
+    "watch:app": "NODE_ENV=development nodemon -x 'npm run hof-transpile && npm start' --ignore 'apps/**/translations/en/'",
     "watch:scss": "nodemon -e scss -x 'npm run sass'",
     "watch:js": "nodemon --watch assets/js -e js -x 'npm run browserify'",
     "watch:translations": "nodemon --watch apps/**/translations/src -x 'npm run hof-transpile'",
```

Comment:
it's not nodeman that parses the argument, hof-transpile does. 
Yeah, it works, but if it's redundnat now, meh...